### PR TITLE
Remove transform scale on left menu button click

### LINKED ISSUE #1609: Transition transform effect removal on left hand menu items
Remove the transform transition on click. Replace with a "pressed" state (pressed state is currently the same as active state).

[Figma link](https://www.figma.com/design/RGRSmIC4KoVlIosQB5YmQY/%F0%9F%93%B1%F0%9F%A7%B1-App-components?m=auto&node-id=971-5497&t=vTR45Pqi1V8KWQrh-1)

Image

### PR DESCRIPTION
**Pull request by insomnious**
@captainsandypants is happy with the global  removal of transform scale, and I've commented out the press state as it doesn't make sense until the active state is implemented

Fixes #1609

### CODE REVIEW

**Review by Sewer56 — COMMENTED** [review]
This change will also affect the Spine. Is that intended?

We might want to go through uses of `Property="RenderTransform" Value="scale(`, and selectively remove the ones we don't want for now.

Rest is fine to me.

**Comment by insomnious** [comment]
The spine buttons are set specifically in their style and so they aren't affected by the global `<Setter Property="RenderTransform" Value="none" />` anyway. 

Let me add a commit removing the other localised scale resets like in:

image

**Comment by Sewer56** [comment]
> The spine buttons are set specifically in their style 

Yeah, seems like it.
I was looking at the download button earlier, which didn't have that.

Seems to be a bug, as it scaled differently than the rest. I fixed that on my branch, so we're good.

File: src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Button/PillStyles.xaml [hunk 1]
```diff
@@ -219,12 +219,8 @@
 
     <!-- Pill Label -->
     <!-- Labels should have all interactions disabled, so they don't look interactable -->
+    <!-- NOTE(insomnious): left this empty selector here to make it easier to find in future -->
     <Style Selector="Button.Pill.Label">
-
-        <!--  Disable the scaling found in Fluent buttons when pressing -->
-        <Style Selector="^:pressed">
```
Comment (Sewer56) [inline src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Button/PillStyles.xaml:225]: This should probably stay.
The pills, like `Premium` should be non-interactable generally. 
By removing the 1.0, Avalonia applies the default, which is to shrink.

File: src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Button/BareIconStyles.axaml [hunk 2]
```diff
@@ -43,11 +43,6 @@
             <Setter Property="Size" Value="24" />
         </Style>
 
-        <Style Selector="^:pressed">
-            <!-- Accentuate the pressed animation -->
```
Comment (Sewer56) [inline src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Button/BareIconStyles.axaml:47]: This is used in the downloads (In Progress) page; and should not be removed.

**Review by Sewer56 — CHANGES_REQUESTED** [review]
(See Above)

Anyway I tested the rest and it's good to me.
So just get those two undone and we're good.

File: src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Button/BareIconStyles.axaml (hunk 2 above)
Comment (insomnious) [reply src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Button/BareIconStyles.axaml:47]: Requested by @captainsandypants, Updated with color states so we don't have to use transform scale

https://github.com/Nexus-Mods/NexusMods.App/assets/26166085/3cc7ea3f-7b57-4273-a085-c055f3a9f6ab

**Review by Sewer56 — APPROVED** [review]

**Pull request merged**